src/components/NewLocationPage/Shared/Shared.style.tsx

```diff
@@ -5,7 +5,7 @@ import { COLOR_MAP } from 'common/colors';
 
 export const SectionContainer = styled.div`
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
```

Comment:
lets avoid editing the styles of reusable parent div. instead, you can add a wrapping div with `display:flex` inside of `SectionContainer`